src/NzbDrone.Core/Indexers/IIndexerSettings.cs

```diff
@@ -5,5 +5,6 @@ namespace NzbDrone.Core.Indexers
     public interface IIndexerSettings : IProviderConfig
     {
         string BaseUrl { get; set; }
+        int Priority { get; set; }
```

Comment:
@markus101 I didn't think about this a few days ago, but this should not be a Field at all! If it's global for all indexers, it belongs straight in IndexerDefinition (and yes, that would require a db migration and that also means it needs to be added to the UI separately.)